packages/core/demo/data/line.ts

```diff
@@ -228,7 +228,10 @@ export const lineTimeSeriesCustomDomainOptions = {
 			title: "2019 Annual Sales Figures",
 			domain: [new Date(2019, 0, 5), new Date(2019, 0, 15)],
 			mapsTo: "date",
-			scaleType: "time"
+			scaleType: "time",
+			ticks: {
+				values: [new Date(2019, 0, 11)]
+			}
```

Comment:
do we want to keep this?

Reply:
yeah we dont need to keep this - ill remove this